janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -148,9 +82,7 @@
     final ExecutorService executorService;
 
     @Resource
-    private Cluster cluster;
-    @Resource
-    private Session session;
+    private final CqlSession session;
```

Comment:
@porunov Think I finally now understand the reason why some of the tests are failing with the Mockito exception
```
org.mockito.exceptions.misusing.UnnecessaryStubbingException
```
And it's because the `session` member variable was made `final` and this prevents the `@InjectMocks` annotation in `CQLStoreTest` from injecting the mock session instance which the tests are expecting. When I remove the `final` qualifier from the session the tests pass for me.